### PR TITLE
Add runtime configuration section to readme

### PR DESCRIPTION
**Pull request by klingerf**
This branch updates the README with info about docker-specific environment variables that can be set to control a running docker container. I've also included links back to the Zipkin documentation of other available environment variables where appropriate. And I've added information about the minimum versions of docker / docker-compose required to run the v2 docker-compose files. I'm not super familiar with how all of the containers are configured, so please correct if I've made any mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,18 +57,73 @@ To start the legacy configuration, run:
...
+
+For the query and web containers, if `TRANSPORT_TYPE=http`, then those
+containers will send trace data via http to the `zipkin-query` service. If
+`TRANSPORT_TYPE=scribe`, then  those containers will send trace data via scribe
+to the `zipkin-collector` service. If `TRANSPORT_TYPE` is unset, then those
+containers will not trace requests that they receive.
+
+For the collector container, if `TRANSPORT_TYPE=scribe`, then the container will
+run a scribe collector on port 9410. If `TRANSPORT_TYPE=kafka`, then the
+container will poll Kafka, and expects one of these two additional environment
```
Comment (adriancole) [inline README.md]: will also poll Kafka (scribe is still enabled)

**Comment by adriancole** [comment]
very good!

File: README.md (hunk 1 above)
Comment (klingerf) [inline README.md]: Good catch -- fixed in the latest commit.

**Comment by klingerf** [comment]
Ok, made a few more wording tweaks to help to clarify some stuff in 1e1d20bf. After that's reviewed I think this should be good to go.

**Comment by adriancole** [comment]
looks great

**Pull request merged**